README.md

```diff
@@ -24,13 +28,18 @@ Please add your project below if it uses this infrastructure for translations:
 
 ## Installation
 
-Run
+- ### [mdbook-i18n-helpers](./i18n-helpers/README.md):
+
+  Run
 
-```shell
-$ cargo install mdbook-i18n-helpers
-```
+  ```shell
+  $ cargo install mdbook-i18n-helpers
+  ```
 
-Please see [USAGE](USAGE.md) for how to translate your `mdbook` project.
+  Please see [USAGE](./i18n-helpers/USAGE.md) for how to translate your
+  [mdbook](https://github.com/rust-lang/mdBook/) project.
+
+---
```

Comment:
Avoid extra lines before sections, GitHub will already style the heading somehow.

```suggestion
```